science/pymol/Portfile

```diff
@@ -8,7 +8,7 @@ PortGroup           boost 1.0
 github.setup        schrodinger pymol-open-source 3.1.0 v
 github.tarball_from archive
 name                pymol
-revision            0
+revision            1
```

Comment:
if a `revision` increase is needed here it should also be done in the `pymol-devel` subport in this Portfile

Reply:
Thanks for spotting that, @reneeotten. 
Done with [72c83a7](https://github.com/macports/macports-ports/pull/27710/commits/72c83a7a2d7063b88357853be97a6626cac8335f)